frontend_app_logbook/src/component/Logbook.jsx

```diff
@@ -275,23 +360,19 @@ export class Logbook extends React.Component {
             })}
           >
             <div>
-              {(state.logbook.entries.filter(e => {
-                const descriptionElement = document.getElementById(`${e.id}_description`)
-                const descriptionHeight = (descriptionElement || { scrollHeight: 0 }).scrollHeight
-                return descriptionHeight > heightBeforeSeeMoreButton
-              }).length >= 1) && (
+              {(state.logbook.entries.filter(e => e.expand !== DESCRIPTION_BUTTON.HIDDEN).length >= 1) && (
                 <>
                   <IconButton
-                    customClass='logbook__expand_button'
-                    text={props.t('Expand all event')}
-                    textMobile={props.t('Expand all event')}
-                    onClick={() => this.handleExpand()}
+                    customClass={classnames('btn-link', 'logbook__expand_button')}
+                    text={props.t('Expand all event descriptions')}
+                    textMobile={props.t('Expand all event descriptions')}
+                    onClick={() => this.handleExpandAll()}
```

Comment:
you can pass the handler by reference here

Reply:
done